main.tf

```
@@ -1 +1,59 @@
+resource "random_password" "this" {
+  length      = 128
+  lower       = true
+  upper       = true
+  numeric     = true
+  special     = true
+  min_lower   = 1
+  min_upper   = 1
+  min_numeric = 1
+  min_special = 1
+}
 
+resource "azurerm_mysql_flexible_server" "this" {
+  name                         = var.server_name
+  location                     = var.location
+  resource_group_name          = var.resource_group_name
+  version                      = var.server_version
```

Could shorten this according to naming convention (unless you think the longer name is more descriptive?):

```suggestion
  version                      = var.version
```